src/server/game/Maps/Map.cpp

```diff
@@ -2797,6 +2797,7 @@ InstanceMap::~InstanceMap()
 {
     delete instance_data;
     instance_data = nullptr;
+    sInstanceSaveMgr->DeleteInstanceSaveIfNeeded(GetInstanceId(), true);
```

Comment:
@Kitzunu https://github.com/azerothcore/azerothcore-wotlk/commit/f9d708b45005cf6327e3a73d2062a2f92bbe9de6#diff-cacd2443c0f46ce614d849924bdb64dd81aa4d5068b131e1c71ecac518b1c3e1L2615
Why this line has been removed?